### PR TITLE
Deprecations utils.js

### PR DESCRIPTION
**Pull request by Fawke**
## Type of change
- [X] Other

## Description of change

Deprecated functions in `src/utils.js`
- `getTopWindowLocation()`
- `getTopFrameReferrer()`
- `getAncestorOrigins()`
- `getTopWindowUrl()`
- `getTopWindowReferrer()`

Test cases have been updated to reflect the change. 

Adapters which were using this deprecated functions have been removed. 

## Please review but don't merge this time.

### CODE REVIEW

**Comment by mkendall07** [comment]
This pull request **fixes 5 alerts** when merging 3b5f3fae41c24f26bac8a5c1644de921a7d6b3e3 into 8855c1e5e9bda275557ca40d420332e42fa1c2c3 - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-7d9ddd207b5037d43e966dca2e3ef3425f415abd)

**fixed alerts:**

* 3 for Useless conditional
* 1 for Useless assignment to local variable
* 1 for Superfluous trailing arguments

---

**Warning** - Automated code review for prebid/Prebid.js will be disabled on October 1, 2019. You can avoid this by <a href="https://github.com/apps/lgtm-com/installations/new">installing the LGTM.com GitHub App</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by idettman** [comment]
Can you please update with changes from master?

**Comment by anand-venkatraman** [comment]
PulsePoint adapter updated to remove usage of deprecated function #4257

**Comment by idettman** [comment]
@Fawke can you update and resolve the conflicts

**Comment by Fawke** [comment]
> @Fawke can you update and resolve the conflicts

Resolved!

**Comment by Fawke** [comment]
@idettman Would you be able to help me resolve an issue with RubiconBidAdapter_spec.js file? After my last sync with the master branch, one of the test cases in that file is failing, precisely this test. https://github.com/prebid/Prebid.js/blob/34b7d72d8cd88d5a86cc807bb5e378576b3e4258/test/spec/modules/rubiconBidAdapter_spec.js#L474

I tried fix the test, but I wasn't able to. So, right now, I've skipped this test to make the build successful. Would you be able to help me figure out what's wrong with the test, or how to make it pass?

File: test/spec/modules/rubiconBidAdapter_spec.js [hunk 1]
```diff
@@ -476,7 +476,7 @@ describe('the rubicon adapter', function () {
           expect(request.data).to.not.contain('x_source.pchain');
         });
 
-        it('ad engine query params should be ordered correctly', function () {
+        it.skip('ad engine query params should be ordered correctly', function () {
```
Comment (jaiminpanchal27) [inline test/spec/modules/rubiconBidAdapter_spec.js]: Any reason to skip this test ?

Comment (Fawke) [reply test/spec/modules/rubiconBidAdapter_spec.js]: @jaiminpanchal27 Previously, this test case was failing due to some unknown reason, but, we verified it again and it seems to be passing. Have added this test back in my latest commit.

**Comment by Fawke** [comment]
> @idettman Would you be able to help me resolve an issue with **RubiconBidAdapter_spec.js** file? After my last sync with the **prebid-3.0** branch, one of the test cases in that file is failing, precisely this test.
> 
> https://github.com/prebid/Prebid.js/blob/34b7d72d8cd88d5a86cc807bb5e378576b3e4258/test/spec/modules/rubiconBidAdapter_spec.js#L474
> 
> I tried fix the test, but I wasn't able to. So, right now, I've skipped this test to make the build successful. Would you be able to help me figure out what's wrong with the test, or how to make it pass?

@idettman Nevermind, the test seems to be passing now.

**Pull request merged**